app/views/email_notifications/edit.njk

```diff
@@ -20,12 +20,11 @@ Change email notifications template - {{currentServiceName}} {{currentGatewayAcc
   <ul class="list-bullet push-bottom">
     <li>let the user know if they need to do something else to complete the service or application</li>
     <li>include information regarding delivery dates or how long it will take to process the payment</li>
-    <li>link to a page on GOV.UK if you want to provide contact service details</li>
-    <li>spell out any web addresses (URLs) in full to show the user where links are going - for example, https://www.gov.uk/exampleservice</li>
-    <li>URLs should include HTTPS</li>
-    <li>avoid including redirects in any links - for example, tracking</li>
-    <li>don’t link directly to a sign in page</li>
-    <li>don’t request personal data passwords or payment details</li>
+    <li>don’t request personal data, passwords or payment details</li>
+    <li>link to a page on GOV.UK if you want to provide details to contact the service</li>
+    <li>don’t link directly to a sign-in page</li>
+    <li>don’t use redirects or tracking links</li>
+    <li>links must use https where available, point to a gov.uk domain, and show the url in full. For example: apply now at https://www.gov.uk/example</li>
```

Comment:
I know this matches the text in the ticket but “https” and “url” should be “HTTPS” and “URL” respectively